results/intfloat__multilingual-e5-small/RestaurantReviewSentimentClassification.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dataset_revision": "d51bf2435d030e0041344f576c5e8d7154828977",
+  "mteb_dataset_name": "RestaurantReviewSentimentClassification",
+  "mteb_version": "1.6.3",
+  "test": {
```

Comment:
Results seem okay, but I wonder how it worked on the `test` split although only a `train` split is available on HF?